piece_of_cake.py

```diff
@@ -0,0 +1,14 @@
+# Arguments:
+# prices - dictionary[key: item's name, value: price for 100g]
+# optionals - list of ingredients to ignore.
+# ingredients - dictionary[ingredient_name : amount in grams]
+# Return value - float
+# This functions calculates the total price for wanted ingredients
+def get_recipe_price(prices, optionals=[], **ingredients):
+
+    if not ingredients:
+        return 0
+
+    return sum(prices[ingredient] / 100 * amount
+               for ingredient, amount in ingredients.items()
+               if ingredient not in optionals)
```

Comment:
Where is an example of this executed code? inside a https://www.freecodecamp.org/news/if-name-main-python-example/

Reply:
what?